src/java/org/apache/cassandra/index/sai/disk/v1/MetadataWriter.java

```diff
@@ -43,17 +43,17 @@ public Builder builder(String name)
         return new Builder(name);
     }
 
-    public class Builder extends RAMIndexOutput implements Closeable
+    public class Builder extends ResettableByteBuffersIndexOutput implements Closeable
     {
         private Builder(String name)
         {
-            super(name);
+            super(1024, name);
```

Comment:
Curious about why 1024 made the most sense here...

Reply:
I ... don't remember.  I put it back to 128 like it was before and updated CASSANDRA-18280 to point to the new code.